cluster/syncer/statefulset.go

```diff
@@ -232,7 +242,10 @@ func (s *StatefulSetSyncer) updatePod(ctx context.Context) error {
 	return nil
 }
 
-// preUpdate run before update the leader pod when replica is 2.
+// preUpdate run before update the leader pod when replicas is 2.
+// For a two-node cluster, we need to adjust the parameter rpl_semi_sync_master_timeout 
```

Comment:
Wrong describe. `rpl_semi_sync_master_timeout` is automatically set by xenon.
`preUpdate` used to manually switch the master node.